### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by Artistic18**

### CODE REVIEW

**Comment by Artistic18** [comment]
Please Review

**Comment by matthewsamuel95** [comment]
amazing!

File: README.md [hunk 1]
```diff
@@ -24,15 +24,150 @@ ACM-ICPC Algorithms is a collection of important algorithms and data structures
...
+   * [Longest Common Subsequence](/DP/LongestCommonSubsequence)
+   * [Longest Increasing Subsequence](/DP/LongestIncreasingSubsequence)
+   * [Longest Repeated Subsequence](/DP/Longest%20Repeated%20Subsequence)
+   * [Matrix Chain Multiplication](/DP/MatrixChain_multiplication)
+   * [Max Sum Increasing Subsequence](/DP/Maximum%20Sum%20Increasing%20Subsequence)
+   * [Minimum Path Sum](/DP/MinimumPathSum)
+   * [Number Of Islands](/DP/NumberOfIslands)
+   * [Print Neatly](/DP/PrintNeatly)
+   * [Shortest Uncommon Subsequence](/DP/ShortestUncommonSubsequence)
+   * [Subset Sum](/DP/subset%20sum%20problem)
```
Comment (U-ways) [inline README.md]: Change to `/DP/subset_sum%20problem`.

Comment (U-ways) [reply README.md]: @matthewsamuel95 I guess I was 13 minutes too slow haha (Simply fix regardless)

Comment (matthewsamuel95) [reply README.md]: 👏

File: README.md [hunk 2]
```diff
@@ -24,15 +24,150 @@ ACM-ICPC Algorithms is a collection of important algorithms and data structures
...
+   * [Last Digit Exp](/Math/last_digit_exp)
+   * [Logarithm](/Math/log)
+   * [Lowest Common Multiple](/Math/lowest_common_multiple)
+   * [Matrix Power](/Math/Matrix_Power)
+   * [Max Divisible Number](/Math/max_divisible_num)
+   * [Max Sub Rectangle](/Math/max_sub_rectangle)
+   * [Max Sub Square](/Math/Max_Sub_Square)
+   * [Miller Rabin Primality Test](/Math/miller_rabin_primality_test)
+   * [Modular Multiplication Inverse](/Math/modular_multiplicative_inverse)
+   * [Nest Power of 2](/Math/NextPow2)
```
Comment (matthewsamuel95) [inline README.md]: typo lol should be next

File: README.md (hunk 1 above)
Comment (Artistic18) [reply README.md]: :)

**Pull request merged**